packages/prisma/schema.prisma

```diff
@@ -249,19 +250,20 @@ model VerificationToken {
 }
 
 model BookingReference {
-  id                 Int      @id @default(autoincrement())
+  id                 Int         @id @default(autoincrement())
   /// @zod.min(1)
   type               String
   /// @zod.min(1)
   uid                String
   meetingId          String?
   meetingPassword    String?
   meetingUrl         String?
-  booking            Booking? @relation(fields: [bookingId], references: [id], onDelete: Cascade)
+  booking            Booking?    @relation(fields: [bookingId], references: [id], onDelete: Cascade)
   bookingId          Int?
   externalCalendarId String?
   deleted            Boolean?
   credentialId       Int?
+  credential         Credential? @relation(fields: [credentialId], references: [id])
```

Comment:
Since `credentialId` is already present, connected credentials appear in the DB.